packages/SwingSet/src/kernel/vatAdmin/vatAdmin-src.js

```diff
@@ -32,9 +32,17 @@ export default function setup(syscall, state, helpers, endowments) {
     return harden({
       // Called by the wrapper vat to create a new vat. Gets a new ID from the
       // kernel's vat creator fn. Remember that the root object will arrive
-      // separately. Clean up the outgoing and incoming arguments.
+      // separately. We clean up the outgoing and incoming arguments.
       create(code) {
-        return kernelVatCreationFn(`${code}`);
+        const bundle = harden({
```

Comment:
At some point (not necessarily with this commit), could you add a doc to `docs/dynamic-vats.md` and describe the shape of the source code for the new vat at various points along its delivery? I'd love to see a minimal example of the string which this `create()` method expects to receive, followed by what the kernel function expects to receive, etc. I suspect this pathway will be fragile under maintenance, and more docs may help us keep it working correctly.

Reply:
I'll add something shortly.